config.yaml

```diff
@@ -13,3 +13,20 @@ options:
     type: string
     default: "127.0.0.1"
     description: "listen address"
+  ssl_cert:
+    type: string
+    default:
+    description: |
+      base64-encoded SSL certificate to install and use for nginx site.
+  ssl_key:
+    type: string
+    default:
+    description: |
+      base64-encoded SSL key to use with certificate specified as ssl_cert.
+  ca_cert:
```

Comment:
Replace ssl_ca with ca_cert to maintain consistency with the charm configuration where ssl_ca is not used.

Reply:
with which charm configuration? `ssl_{ca,cert,key}` is a somewhat established (legacy) pattern, see for example the keystone charm

Reply:
In the charm code is used ca_cert but not ssl_ca, I will update it to ssl_ca.